packages/webdriverio/src/scripts/isElementDisplayed.js

```diff
@@ -152,12 +157,21 @@ export default function isElementDisplayed(element) {
             return isElementSubtreeHiddenByOverflow(childNode)
         })
     }
+    // walk up the tree testing for a shadow root
+    function isElementInsideShadowRoot(element) {
+        if (!element) {
```

Comment:
Can element  be undefined at this point?

Reply:
I think it can be null. Are you questioning the need for the conditional of suggesting it is more explicit?

Reply:
If it can be null, I'm fine with it, thank you